ivi/interface/pyvisa.py

```diff
@@ -112,20 +109,20 @@ def trigger(self):
 
     def clear(self):
         "Send clear command"
-        raise NotImplementedError()
+        self.instrument.clear()
 
     def remote(self):
         "Send remote command"
-        raise NotImplementedError()
+        self.instrument.gpib_control_ren(visa.constants.VI_GPIB_REN_ASSERT_ADDRESS)
 
```

Comment:
I would recommend using the `control_ren` method instead, as this should be generic enough across different types of VISA connections.

For reference, see `ControlRenMixin` in https://github.com/hgrecco/pyvisa/blob/898b948ec835f3cac8ee4aaec9317237ad5a31bd/pyvisa/resources/messagebased.py